src/netif.c

```diff
@@ -1263,6 +1263,21 @@ static void netif_get_isol_rx_lcores(uint8_t *nb, uint64_t *mask)
         *mask = isol_lcore_mask;
 }
 
+static void build_lcore_index(int lcore_max)
+{
+    int idx = 0, ii;
+
+    g_lcore_index[idx++] = rte_get_master_lcore();
+
+    for (ii = 0; ii < lcore_max; ii++)
+        if (g_lcore_role[ii] == LCORE_ROLE_FWD_WORKER)
+            g_lcore_index[idx++] = ii;
+
+    for (ii = 0; ii < lcore_max; ii++)
+        if (g_lcore_role[ii] == LCORE_ROLE_ISOLRX_WORKER)
+            g_lcore_index[idx++] = ii;
+}
```

Comment:
I think one ‘for’ is enough. Why not use 'g_lcore_role[ii] == LCORE_ROLE_FWD_WORKER || g_lcore_role[ii] == LCORE_ROLE_ISOLRX_WORKER'

Reply:
The array elems of  `g_lcore_index[]` are arranged with designated order as follow:
g_lcore_index[0]                <-- master lcore
g_lcore_index[1..n]            <-- fowarding lcores
g_lcore_index[n+1, n+m]  <-- isol_rx lcores
The two loops cannot be combined.